src/game.cpp

```diff
@@ -3366,6 +3371,10 @@ void Game::processPlayerInteraction(std::vector<aabb3f> &highlight_boxes,
 			playeritem = mlist->getItem(client->getPlayerItem());
 	}
 
+	if (playeritem.getDefinition(itemdef_manager).name.empty()) { // override the hand
+		if (InventoryList *hlist = local_inventory->getList("hand"))
```

Comment:
Yeah, I'm not fine with this. It requires me to look up the C++ standard to understand the semantics.... that aint good.